lib/smack/SmackRep.cpp

```diff
@@ -514,14 +516,27 @@ const Stmt* SmackRep::returnValueAnnotation(const CallInst& CI) {
 //
 // }
 
+bool SmackRep::isUnsafeFloatAccess(const Type* elemTy, const Type* resultTy) {
+  if (elemTy->isFloatingPointTy()) {
```

Comment:
I think I already mentioned this, but why don't you merge these into just one if condition?